shared/actions/teams.js

```diff
@@ -375,6 +375,17 @@ const _getDetails = function*(action: TeamsGen.GetDetailsPayload): Saga.SagaGene
       team: publicity.teamShowcase.isShowcased,
     }
 
+    let operationsMap: Map<string, boolean> = new Map()
```

Comment:
Might be better to use a plain object here. We typically 99% of the time use Immutable.Map and not the es6 Map

Reply:
Honestly it might be better to just change the rpc to just give us this map back instead of us having to call it over and over if this is our only use case